code/go/0chain.net/chaincore/chain/protocol_block.go

```diff
@@ -490,6 +469,27 @@ func (c *Chain) finalizeBlock(ctx context.Context, fb *block.Block, bsh BlockSta
 		}
 	}
 
+	wg.Run("finalize block - delete dead blocks", fb.Round, func() error {
```

Comment:
We'd better use `wg.Run` together with `wg.Wait()`, or we can simply run the function in a goroutine if we don't need to wait for it.